web/management/commands/create_test_data.py

```
@@ -575,3 +637,307 @@ def handle(self, *args, **kwargs):
             self.stdout.write(f"Created images for product: {product.name}")
 
         self.stdout.write(self.style.SUCCESS("Successfully created test data"))
+
+    def create_exams_and_quizzes(self, courses: list, sessions: list, students: list) -> None:
```

Ruff (0.8.2)

`create_exams_and_quizzes` is too complex (15 > 10)

(C901)